modules/delegates/components/DelegateOverviewCard.tsx

```diff
@@ -322,17 +337,19 @@ export function DelegateOverviewCard({ delegate }: PropTypes): React.ReactElemen
           delegate={delegate}
           isOpen={showDelegateModal}
           onDismiss={() => setShowDelegateModal(false)}
-          mutateTotalStaked={mutateTotalStaked}
+          mutateTotalStaked={mutateDelegateTotalMkr}
           mutateMKRDelegated={mutateMKRDelegated}
+          refetchOnDelegation={false}
         />
       )}
       {showUndelegateModal && (
         <UndelegateModal
           delegate={delegate}
           isOpen={showUndelegateModal}
           onDismiss={() => setShowUndelegateModal(false)}
-          mutateTotalStaked={mutateTotalStaked}
+          mutateTotalStaked={mutateDelegateTotalMkr}
           mutateMKRDelegated={mutateMKRDelegated}
```

Comment:
What is the difference between `mutateTotalStaked` and `mutateMKRDelegated`? They seem like they would serve the same purpose.

And do you know the reason why `refetchOnDelegation` is sometimes `true` and sometimes `false`?